### PR TITLE
Add codedark theme

### PR DESCRIPTION
**Pull request by Shatur95**
normal
insert
visual
replace

Ported from [vim-code-dark](https://github.com/tomasiser/vim-code-dark) using [lightline2lualine_theme_converter.lua](https://gist.github.com/shadmansaleh/000871c9a608a012721c6acc6d7a19b9).

### CODE REVIEW

**Comment by hoob3rt** [comment]
Onedark already exists. Is the current implementation not inline with how the theme is supposed to work?

**Comment by Shatur95** [comment]
> Onedark already exists.

This is Code dark (VSCode default theme), not related to One dark (Atom default theme).

**Comment by hoob3rt** [comment]
Oh yea yea, my bad. I misread it

File: THEMES.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All available themes are only best effort ports by myself/ other users. If you find a theme to be weird/ wrong please open an issue/ pr.
 
+### onedark
```
Comment (hoob3rt) [inline THEMES.md:5]: should be changed to codedark then

Comment (Shatur95) [reply THEMES.md]: Thanks, renamed.

**Comment by hoob3rt** [comment]
Screenshot 2021-02-04 at 21 13 43

also, I've tested it with [this version of codedark for vim](https://github.com/tomasiser/vim-code-dark) and the blue seems a bit off. Am I missing something here?

**Comment by Shatur95** [comment]
> also, I've tested it with this version of codedark for vim and the blue seems a bit off. Am I missing something here?

Hm... Looks ok to me. But do you want to suggest another color?

**Comment by hoob3rt** [comment]
Well I guess fair enough. It looks all right for me and if you, a user of codedark, approve it then it's okay

**Comment by Shatur95** [comment]
Thank you!

**Pull request merged**